recipes/tttrlib/build.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+mkdir b2 && cd b2
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)